geometry_msgs/PoseStamped ps
float64 wait_time